lib/provisioner/worker/plugins/automators/chef_solo_automator/resources/cookbooks/coopr_dns/recipes/dnsimple.rb

```diff
@@ -17,34 +17,6 @@
 # limitations under the License.
 #
 
-# Install some pre-requisites
-case node['platform_family']
-when 'debian'
-  zpkg = 'libz-dev'
-when 'rhel'
-  zpkg = 'zlib-devel'
-end
-
-r = package( zpkg ) { action :nothing }
-r.run_action( :install )
-
-%w(build-essential dnsimple).each do |recipe|
-  include_recipe recipe
```

Comment:
hopefully nothing downstream is depending on build-essential being included already